WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/Edit Inventory/ProductInventorySettingsViewModelTests.swift

```diff
@@ -141,7 +141,7 @@ final class ProductInventorySettingsViewModelTests: XCTestCase {
         XCTAssertEqual(isSKUValid, false)
         XCTAssertEqual(shouldBringUpKeyboard, true)
         let expectedSections: [Section] = [
-            .init(errorTitle: ProductUpdateError.duplicatedSKU.alertMessage, rows: [.sku]),
+            .init(errorTitle: ProductUpdateError.duplicatedSKU.localizedDescription, rows: [.sku]),
```

Comment:
```suggestion
            .init(errorTitle: ProductUpdateError.duplicatedSKU.errorDescription, rows: [.sku]),
```

Reply:
yup! updated in https://github.com/woocommerce/woocommerce-ios/pull/2781/commits/1b36374546a33d4941e6024156bd2e9b821525e1